pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -70,7 +71,12 @@ var (
 		// this value to 3, however that has shown to be insufficient during periods of pod churn (e.g. rolling updates),
 		// where multiple endpoints in a cluster are terminated. In these scenarios the circuit breaker can kick
 		// in before Pilot is able to deliver an updated endpoint list to Envoy, leading to client-facing 503s.
-		MaxRetries: &wrappers.UInt32Value{Value: 1024},
+		MaxRetries: &wrappers.UInt32Value{Value: math.MaxUint32},
```

Comment:
dont you need to set the same for inbound as well?

Reply:
I don't think the inbound clusters were causing issues as the default connection pool sizes are pretty generous when applied per-pod, but I agree that if we disable it, we should disable completely. I'm now applying the same defaults on inbound clusters